docs/wmco-prerequisites.md

```diff
@@ -5,13 +5,12 @@ may be relevant.
 ## Supported Cloud Providers based on OKD/OCP Version and WMCO version
 | Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
 |---------------------------|---------------------------|------------------------|
-| Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
 | VMware vSphere            | 4.7+                      | WMCO 2.0+              |
 | Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
 
-Note: We added Azure support in 4.6 but given that Microsoft has [stopped publishing Windows Server 2019 images with
+Note: We added Azure and AWS support in 4.6 but given that Microsoft and AWS have [stopped publishing Windows Server 2019 images with
```

Comment:
```suggestion
Note: We added Azure and AWS support in 4.6, but because Microsoft and AWS have [stopped publishing Windows Server 2019 images with
```